### ISSUE TITLE
Whitelist test fails

### ISSUE THREAD

**Issue by astorije** [dontcallmedom/third-party-resources-checker#3]
Running the test suite with a fresh clone currently outputs:

```bash
$ npm test

> third-party-resources-checker@0.0.1 test [...]/third-party-resources-checker
> mocha

  ․․․․․․․․․

  8 passing (2s)
  1 failing

  1) Starting test suite should ignores whitelisted 3rd party url:

      Uncaught Error: expected [ 'http://example.org/' ] to sort of equal []
      + expected - actual

      +[]
      -[
      -  "http://example.org/"
      -]
      
      at Assertion.assert ([...]/third-party-resources-checker/node_modules/expect.js/index.js:96:13)
      at Assertion.eql ([...]/third-party-resources-checker/node_modules/expect.js/index.js:230:10)
      at ChildProcess.<anonymous> ([...]/third-party-resources-checker/test/mocha.js:74:39)
      at ChildProcess.EventEmitter.emit (events.js:98:17)
      at maybeClose (child_process.js:743:16)
      at Process.ChildProcess._handle.onexit (child_process.js:810:5)

npm ERR! Test failed.  See above for more details.
```

**Comment by dontcallmedom** [comment]
closed via 17f2590c75 ; also, Travis CI is now set up to avoid this type of errors
https://travis-ci.org/dontcallmedom/third-party-resources-checker/

**Comment by astorije** [comment]
Argh, stupid me, I should have sent this outside of #4 :(
I forgot that the PR was including this fix when I put it on hold, sorry!
